consultation_analyser/consultations/llm_summariser.py

```diff
@@ -0,0 +1,25 @@
+"""
+Use LLMs to generate summaries for themes. More to follow!
+"""
+from django.conf import settings
+
+from consultation_analyser.consultations.decorators.sagemaker_endpoint_status_check import check_and_launch_sagemaker
+from consultation_analyser.consultations.models import Theme
+
+
+def dummy_generate_theme_summary(theme: Theme) -> str:
+    made_up_summary = (", ").join(theme.keywords)
+    return made_up_summary
+
+
+@check_and_launch_sagemaker
```

Comment:
Tidy!

Reply:
Maybe the decorator could pass an llm object into the function, which would be sagemaker if `USE_SAGEMAKER_LLM` and something else if not?

Reply:
Can I make this change as part of the next PR which is dealing with the actual LLM stuff - in the interests of not making this PR too giant (and in the next PR I'll have more of an idea of how I'm structuring the LLM code).